framework/routing/__init__.py

```diff
@@ -30,6 +30,16 @@
     ],
     module_directory='/tmp/mako_modules'
 )
+
+_tpl_lookup_safe = TemplateLookup(
```

Comment:
As discussed in person, this might not be necessary. Worth looking into whether we need this